lib/fortnox/api/models/document_base.rb

```diff
@@ -0,0 +1,183 @@
+require "fortnox/api/models/base"
+require "fortnox/api/models/email_information"
+
+module Fortnox
+  module API
+    module Model
+      module DocumentBase
+        #rubocop:disable Metrics/MethodLength
+        def self.included(base)
+          # Url Direct url to the record.
```

Comment:
I would add a comment here to document why we decided to do it this way instead of breaking it into several smaller methods. The readability argument in other words.